test/test_auth_token.py

```diff
@@ -99,6 +99,16 @@ def test_should_ignore_url_if_acl_is_provided(self):
     def test_must_provide_expiration_or_duration(self):
         self.assertRaises(Exception, cloudinary.utils.generate_auth_token, acl="*", expiration=None, duration=None)
 
+    def test_must_provide_acl_or_url(self):
+        self.assertRaises(Exception, cloudinary.utils.generate_auth_token, start_time=1111111111, duration=300)
+
+        cloudinary.utils.generate_auth_token(start_time=1111111111, duration=300, acl="/*/t_foobar")
```

Comment:
These to tests (only acl and only url) should be covered separately. Please add tests, if they are not covered, otherwise please remove these lines.